### PR TITLE
fix(README.md): add examples to Git Commit Guidelines in README.md

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Updates the Git Commit Guidelines with a few additional examples to help ensure future PRs can easily conform to desired commit message structure.

PR made in reference to this [comment](https://github.com/patternfly/angular-patternfly/pull/587#issuecomment-324306096) & this [comment](https://github.com/patternfly/angular-patternfly/pull/587#issuecomment-324344667).

@jeff-phillips-18

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
Quick Q, should there be more than `fix`, `perf`, and `feat`?  I see one example in the history:

`
chore(pfToolbar): fix typo in directive name and link from 'componenet' to 'component' 
`

`chore` is not doc'd here.

**Comment by dtaylor113** [comment]
Hi,  ```chore``` will be merged but not update the version at all.  Yeah, I didn't see it documented either, but that is what it does :-)

**Comment by jeff-phillips-18** [comment]
Correct, chore is documented in the link but not here.

This should be a chore so as not to cause a release.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 [![Join the chat at https://gitter.im/patternfly/angular-patternfly](https://badges.gitter.im/patternfly/angular-patternfly.svg)](https://gitter.im/patternfly/angular-patternfly?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-# AngularJS directives for [PatternFly](https://www.patternfly.org) 
+# AngularJS directives for [PatternFly](https://www.patternfly.org)
```
Comment (dtaylor113) [inline README.md]: This should probably be rewritten as "AngularJS components..."

**Comment by benjaminapetersen** [comment]
So something like:

```
##### Non-Release

chore(foo): some minor change that should not trigger an automated release.
```

File: README.md (hunk 1 above)
Comment (benjaminapetersen) [inline README.md]: I can quick update that to `components`.

**Comment by dtaylor113** [comment]
> So something like:
> 
> ##### Non-Release
> 
> chore(foo): some minor change that should not trigger an automated release.

Yes, that would be perfect

**Comment by benjaminapetersen** [comment]
Done

**Pull request merged**